### ISSUE TITLE
[Ze] Save Score Fix.

### ISSUE THREAD

**Issue by lucaslinha** [quasemago/zrageservers#151]
Fixar o save de score no servidor, ter um indicativo de level como esse em cada jogador é de extrema importancia para mapas com leveis nos itens.

**Comment by quasemago** [comment]
Score é usado pelo topdefender.

**Comment by lucaslinha** [comment]
> Score é usado pelo topdefender.

Não vejo problema nenhum perder isso so nos mapas que tem level nas materias, não vai matar ngm.

**Comment by hcjordao** [comment]
Concordo com o infused que não ter o score em alguns mapas não vai matar ninguém.

Acho valido inserir uma mensagem que nem o EW.

[Save Score] está ativo nesse mapa. Devido isso score de top defender estará desativado nesse meio tempo.
